portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -28,16 +28,24 @@
 public class DataServlet extends HttpServlet {
 
   static final String CONTENT_TYPE = "application/json;";
+  final List messages = new ArrayList<String>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    List messages = new ArrayList<String>();
-    messages.add("Nice portfolio!");
-    messages.add("I found the fun stuff ;)");
-    messages.add("Bark!");
     Gson gson = new Gson();
     String json = gson.toJson(messages);
     response.setContentType(CONTENT_TYPE);
     response.getWriter().println(json);
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String comment = request.getParameter("comment").trim();
+    if (comment.length() == 0) {
+      System.err.println("Empty comment submitted");
+      return;
```

Comment:
Please refer to comment about exception handling from another PR. Prefer fix here.

Reply:
Fix implemented here. I left the comment below:

"So I ended up realizing this later in my coding journey. In this PR, they were directed to a blank page, but commit e0f906f will redirect the user no matter what [this is the fix I added, along with sending a 400 error]. Is there a way to let the user know that they were unsuccessful, while still remaining at the comment box?"